docs/user-guide/release-notes/2.12.x.rst

```diff
@@ -9,6 +9,8 @@ New Features
 ------------
 
  * Task profiling can now be enabled. This will use cProfiling on an individual task and write the profile to a directory for a given task. While this can impact performance, this can enable users to get some insight into what a task is doing or use the output to give to a developer for debugging.
+ * When ``pulp-manage-db`` is run, check if there are running processes before continuing. Users
```

Comment:
I'm not sure a user would understand which processes are being waited on. For example we don't wait on `httpd`. Also I hope to identify this works in both standalone and clustered installs. Consider replacing it with a more explicit version like:

`pulp-manage-db will not continue if there are pulp_celerybeat, pulp_resource_manager, or pulp_workers running. This works in both standalone and clustered installations. In some cases, Pulp may wait up to 92 seconds to determine if should continue or not.`